webhook/server/server.go

```diff
@@ -20,13 +20,20 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
+const (
+	InjectEvent          = "Inject"
+	UpdatePodEvent       = "UpdatePod"
+	MissingDynakubeEvent = "MissingDynakube"
```

Comment:
as far as I can see, these constants are not used anywhere but in this package. So they do not need to be exported (capitalized)